lib/resources/accounts.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { normalizeNotes } = require('../utils/razorpay-utils')
+
+module.exports = function (api) {
+
+    const BASE_URL = "/accounts";
+
+    return {
+        create(params, callback) {
+            let { notes, ...rest } = params
+            let data = Object.assign(rest, normalizeNotes(notes));
```

Comment:
What does this `normalizeNotes` do? I saw the implementation, it just adds `notes` to every key?  The curl request doesn't match with this data. 

```
var data = { "phone": "9000090000" } ; 
// gets converted to
var normalised = { "notes[phone]" : "9000090000" } ;
```

Reply:
@1995navinkumar i had checked this one, but notes are not working without `normalizeNotes`. 

**sending notes without normalizeNotes , its throw error**

<img width="623" alt="Screenshot 2023-06-26 at 10 39 27 AM" src="https://github.com/razorpay/razorpay-node/assets/89454448/695d5c0e-67a6-46df-9867-3509d18d5343">


**sending notes with normalizeNotes.** 

<img width="421" alt="Screenshot 2023-06-26 at 10 40 40 AM" src="https://github.com/razorpay/razorpay-node/assets/89454448/47b37daa-4ea2-489e-82ae-58318e358a3f">

Reply:
@ankitdas13  How did you test this? Using a test account? Also, can you share the request payload before and after normalizeNotes?

Reply:
@1995navinkumar yes i used my test amount.  
Payload while using normalizeNotes
<img width="432" alt="Screenshot 2023-06-27 at 5 59 47 PM" src="https://github.com/razorpay/razorpay-node/assets/89454448/b2defb1f-a23b-49ee-9e24-ca65c057c654">

 Payload after sending notes without normalizeNotes 
<img width="390" alt="Screenshot 2023-06-27 at 6 04 33 PM" src="https://github.com/razorpay/razorpay-node/assets/89454448/75c51e14-4d90-4293-97aa-b39d07c8698a">

Reply:
@1995navinkumar Yes, I have a test key,  instead of creating a new object by `Object.assign`, I pass the params to data and it's working without normalizeNotes

```
create(params, callback) {
    return api.post({
        version: 'v2',
        url: `${BASE_URL}`,
        data: params
    }, callback);
},
```
payload 

```
{
  url: '/v2/accounts',
  form: {
    email: 'gauriagain.kum@examples.org',
    phone: '9000090000',
    legal_business_name: 'Acme Corp',
    business_type: 'partnership',
    customer_facing_business_name: 'Example',
    profile: {
      category: 'healthcare',
      subcategory: 'clinic',
      description: 'Healthcare E-commerce platform',
      addresses: [Object],
      business_model: 'Online Clothing ( men, women, ethnic, modern ) fashion and lifestyle, accessories, t-shirt, shirt, track pant, shoes.'
    },
    legal_info: { pan: 'AAACL1234C', gst: '18AABCU9603R1ZM' },
    brand: { color: 'FFFFFF' },
    notes: { internal_ref_id: '123123' },
    contact_name: 'Gaurav Kumar',
    contact_info: { chargeback: [Object], refund: [Object], support: [Object] },
    apps: { websites: [Array], android: [Array], ios: [Array] }
  }
}
```
Should I remove the normalizeNotes from other methods after testing?

Reply:
@1995navinkumar reverted previous changes except client validation